test/e2e/k8s_conformance_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
This file is created by you (a nutanix employee) and not Kubernetes authors

Reply:
its unclear how we want to tag copied files from cluster-api and then modified. Do we have any legal requirements here? @vnephologist

Reply:
added Nutanix Copyright as well